src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -77,6 +77,7 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
   const [applicableFilters, setApplicableFilters] = useState([])
   const [preFilteredSamples, setPreFilteredSamples] = useState([])
   const [stepDisabledByCondition, setStepDisabledByCondition] = useState(false)
+  const [runClicked, setRunClicked] = useState(false);
```

Comment:
since we are using this state only for displaying the modal, what do you think about perhaps calling this `runQCModalVisible` or something like that, that is more direct about what it manages?